include/flatbuffers/flatbuffers.h

```diff
@@ -1800,6 +1800,25 @@ class FlatBufferBuilder {
     return Offset<Vector<const T *>>(EndVector(len));
   }
 
+  /// @brief Serialize an array of native structs into a FlatBuffer `vector`.
+  /// @tparam T The data type of the struct array elements.
+  /// @tparam S The data type of the native struct array elements.
+  /// @param[in] v A pointer to the array of type `S` to serialize into the
+  /// buffer as a `vector`.
+  /// @param[in] len The number of elements to serialize.
+  /// @param[in] packFunc Pointer to a function to convert the native struct
+  /// to the FlatBuffer struct.
+  /// @return Returns a typed `Offset` into the serialized data indicating
+  /// where the vector is stored.
+  template<typename T, typename S>
+  Offset<Vector<const T *>> CreateVectorOfNativeStructs(
+      const S *v, size_t len, T((*const packFunc)(const S &))) {
+    FLATBUFFERS_ASSERT(packFunc);
+    std::vector<T> vv(len);
+    std::transform(v, v + len, vv.begin(), packFunc);
```

Comment:
please use the style of the surrounding code, e.g. `pack_func`

Reply:
renamed the `packFunc` params